guides/7-jlp/2-How-JLP-Works.md

```diff
@@ -93,15 +93,45 @@ hourly borrow fee = (tokens borrowed / tokens in the pool) x 0.01% x position si
 
 The swap fee for the pool typically ranges between 0% and 2%.
 
-### Target Ratio and Fees
+### Pool Weightage
 
-In the JLP pool, every token has a specific target ratio or weightage.
+In JLP, tokens are assigned a **target weight**, which is set by the team. 
 
-The transactions involving the addition or removal of liquidity have the primary purpose of adjusting a token's ratio in the pool to align it more closely with the predefined target. These transactions can be JLP deposits, JLP withdrawals, or token swaps from the JLP pool.
+![jlp-weightage](jlp-weightage.png)
 
-Transactions that move the token's ratio away from the target incur additional fees while instructions that move it closer to the target get a fee discount. The fee is based on the swap fee.
+Experts like [Gauntlet](https://www.gauntlet.xyz/) and [Chaos Labs](https://chaoslabs.xyz/) conduct regular risk assessments and market analyses in response to the evolving market conditions, and discuss with the community before presenting to the team for rollout. 
 
-### Example Yield
+:::tip
+ The risk assessments can be accessed via **[Jupiter's research forum](https://www.jupresear.ch/tag/risk)**.
+:::
+
+### Maintaining Target Weight
+
+The weightages of each asset in JLP will differ based on market activity, particularly Spot Trading or Deposit & Withdrawal of assets into JLP.
```

Comment:
```suggestion
The weightages of each asset in the JLP pool will differ based on market activity, particularly Spot Trading or Deposit & Withdrawal of assets into JLP.
```

Non-blocking; I feel it'd be good to refer to the liquidity pool as the `JLP pool`, because I think `JLP` itself refers to the token. WDYT?